### ISSUE TITLE
Implement new guidelines for optional depictives

### ISSUE THREAD

**Issue by nschneid** [UniversalDependencies/UD_English-EWT#321]
The policy for depictives [has changed](https://universaldependencies.org/changes.html#optional-depictives). A couple of heuristic queries for tokens to review:

- [`acl` edges where the dependent is an ADJ](http://universal.grew.fr/?custom=6275a4035f529)
- [nonprojective `acl` edges where the dependent is a participle](http://universal.grew.fr/?custom=6275a321c5f8e)

**Comment by nschneid** [comment]
Going with enhanced relations `nsubj:depict` (with ADJ) and `nsubj:pass:depict` (with VBN) for now so we don't lose the secondary predication. The naming has yet to be standardized.

**Comment by nschneid** [comment]
In the above commit I've switched to `xcomp` for:

- showing him **complicit**
- I like my food **hot**
- everyone came away **impressed**
- saw it **riddled** with green mold

I think they're more like complements than adjuncts.

**Comment by dan-zeman** [comment]
The [guidelines amendment 4](https://universaldependencies.org/changes.html#optional-depictives) says that "precise naming recommendation for the enhanced edge is deferred for further discussion." Should we put it on the agenda for the core group in November?

Perhaps we should also create a corresponding [issue in the docs repository](https://github.com/UniversalDependencies/docs/issues). Once the naming recommendation is decided, we need to also put it in the [enhanced UD guidelines](https://universaldependencies.org/u/overview/enhanced-syntax.html#controlledraised-subjects). Either as part of the current enhancement "Controlled/raised subjects" (which might then have to be renamed), or as a new, seventh enhancement type.

**Comment by nschneid** [comment]
OK if you think it needs discussion I'll remove the subtype for now. The above commit can be referenced later to restore them.

**Comment by nschneid** [comment]
Either `:xsubj` means a subject implied by `xcomp` specifically, or it means any inferred subject. Could be less confusing to distinguish different kinds of inferred subjects (due to xcomp, depictives, controlled adjuncts). The risk of trying to distinguish them is we would end up with a lot of different labels. But I would lean toward splitting the different subtypes at first, and then it would be easy to merge them later if necessary.

A related question is how much we want to encourage edeprels to be (semi)manual, versus completely automated. Moving beyond conj/relcl/xcomp-generated ones would create a greater need for manual disambiguation, I would think. Not sure how many treebanks are investing effort in this.

**Comment by amir-zeldes** [comment]
I would be against introducing any new subtypes at this point. We have an inflation of rare subtypes already and I also don't see realistic prospects for wide-scale manual annotation of edeps currently. I think we need to concentrate on consolidation rather than introduction of new distinctions before we've shored up the distinctions we just introduced across more than a handful of datasets.

**Comment by nschneid** [comment]
@amir-zeldes you'd prefer to overload `:xsubj`? I don't have a strong opinion TBH.

**Comment by amir-zeldes** [comment]
Yes - I think for datasets that have xsubj there is a good chance that these are already xsubj in other languages as well, so this will promote the most comparability and avoid a very rare deprel as well, which we also don't have a good way of identifying automatically.
